### PR TITLE
Added two speaker blogs and things to do in Nashville

### PR DESCRIPTION
**Pull request by Rittycheriah**

### CODE REVIEW

File: _posts/2019-01-17-things-to-do-in-nashville.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+### Hot Chicken Restaurants:
+* Hattie B's Hot Chicken: Three locations of just pure bliss in fried deliciously spicy chicken
+* Prince's Hot Chicken: The originator of the hot chicken craze in Nashville
+* Bolton's Hot Chicken & Fish: Another fantastic hot fish and chicken restaurant. Fun fact - it was featured on David Chang's Ugly Delicious!
+
+### Unique Museums and Monuments:
+* The Parthenon: A full scale replica of the Parthenon in Greece
+* The Frist Art Museum:  Van Gogh, Monet, Degas, and Their Times: The Mellon Collection of French Art will be the main exhibit the weekend of our conference.
+* Tootsie's and Robert's Western World: Historic country honky tonks: Tootsie's & Robert's.
```
Comment (epochblue) [inline _posts/2019-01-17-things-to-do-in-nashville.md]: Remove one of the "Tootsie's and Robert's", I think?

File: _posts/2019-01-17-things-to-do-in-nashville.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+* Hattie B's Hot Chicken: Three locations of just pure bliss in fried deliciously spicy chicken
+* Prince's Hot Chicken: The originator of the hot chicken craze in Nashville
+* Bolton's Hot Chicken & Fish: Another fantastic hot fish and chicken restaurant. Fun fact - it was featured on David Chang's Ugly Delicious!
+
+### Unique Museums and Monuments:
+* The Parthenon: A full scale replica of the Parthenon in Greece
+* The Frist Art Museum:  Van Gogh, Monet, Degas, and Their Times: The Mellon Collection of French Art will be the main exhibit the weekend of our conference.
+* Tootsie's and Robert's Western World: Historic country honky tonks: Tootsie's & Robert's. 
+* Country Music Hall of Fame: A museum dedicated to the history of country music
+* Lane Motor Museum: The largest European collection of cars and motorcycles in the United States.
```
Comment (epochblue) [inline _posts/2019-01-17-things-to-do-in-nashville.md]: Should this be "The largest collection of European cars and motorcycles..."?

File: _posts/2019-01-17-things-to-do-in-nashville.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+### Vegetarian Friendly Restaurants:
+* Avo: A notably delicious vegan restaurant on the west side of Nashville
+* Chaatable & Chauhan's House of Masala & Ale: Both of these restaurants are owned by the venerable Food Network chef, Maneet Chauhan. Chaatable features Indian street food, and Chauhan's  House of Masala and Ale has a higher end feel with a slightly more fusion-like spin.
+* Sunflower Cafe: A widely vegetarian restaurant with quick eats closer to the conference site
+
+### Tea, Desserts, and more:
+* High Garden: Rated one of the best tea houses in the country
+* The Cupcake Collection: A Germantown instution for almost homemade, classic Southern cupcakes
+* Dozen Bakery: A small bakeshop in the Wedgewood Houston area of town with French classic pastries
```
Comment (epochblue) [inline _posts/2019-01-17-things-to-do-in-nashville.md]: Maybe add a little something to the end here? Just a sentence or two?

File: _posts/2019-01-17-things-to-do-in-nashville.md (hunk 2 above)
Comment (Rittycheriah) [reply _posts/2019-01-17-things-to-do-in-nashville.md]: Yes definitely

File: _posts/2019-01-17-things-to-do-in-nashville.md (hunk 3 above)
Comment (Rittycheriah) [reply _posts/2019-01-17-things-to-do-in-nashville.md]: Oh yeah, a callout to tweet at us maybe about what they visit off or on our list?

File: _posts/2019-01-17-things-to-do-in-nashville.md (hunk 1 above)
Comment (Rittycheriah) [reply _posts/2019-01-17-things-to-do-in-nashville.md]: Ah, weird.

Comment (Rittycheriah) [reply _posts/2019-01-17-things-to-do-in-nashville.md]: Will do.

File: _posts/2019-01-17-things-to-do-in-nashville.md [hunk 4]
```diff
@@ -35,3 +35,5 @@ As a start to beginning your Nashville experience, let us suggest just a couple
 * High Garden: Rated one of the best tea houses in the country
 * The Cupcake Collection: A Germantown instution for almost homemade, classic Southern cupcakes
 * Dozen Bakery: A small bakeshop in the Wedgewood Houston area of town with French classic pastries
+
+Have more Nashville greats we need to add to our list? Did you visit one of our suggestions and want to tell us about it? Tweet us @PyTennessee!
```
Comment (epochblue) [inline _posts/2019-01-17-things-to-do-in-nashville.md]: Add something about tweeting at us if they'd like more/different recommendations, and make `@PyTennessee` a link to https://www.twitter.com/PyTennessee

Comment (Rittycheriah) [reply _posts/2019-01-17-things-to-do-in-nashville.md]: Aye, will do!

**Pull request merged**